workspaces/adoption-insights/plugins/adoption-insights/src/translations/fr.ts

```diff
@@ -0,0 +1,92 @@
+    'header.dateRange.today': "Aujourd'hui",
+    'header.dateRange.lastWeek': 'La semaine dernière',
+    'header.dateRange.lastMonth': 'Le mois dernier',
+    'header.dateRange.last28Days': 'Les 28 derniers jours',
+    'header.dateRange.lastYear': "L'année dernière",
+    'header.dateRange.dateRange': 'Plage de dates...',
+    'header.dateRange.cancel': 'Annuler',
+    'header.dateRange.ok': 'OK',
+    'header.dateRange.defaultLabel': 'Les 28 derniers jours',
+    'activeUsers.title': 'Utilisateurs Actifs',
+    'activeUsers.averagePrefix': 'En moyenne',
+    'activeUsers.averageText': '{{count}} utilisateurs actifs par {{period}}',
+    'activeUsers.averageSuffix': 'étaient actifs au cours de cette période.',
+    'activeUsers.hour': 'heure',
+    'activeUsers.day': 'jour',
+    'activeUsers.legend.newUsers': 'Nouveaux utilisateurs',
+    'activeUsers.legend.returningUsers': 'Utilisateurs récurrents',
+    'templates.title': 'Top modèles',
+    'templates.topNTitle': 'Top {{count}} modèles',
+    'catalogEntities.title': 'Top entités du catalogue',
+    'plugins.title': 'Top plugins',
+    'plugins.topNTitle': 'Top {{count}} plugins',
+    'techDocs.title': 'Top TechDocs',
+    'techDocs.topNTitle': 'Top {{count}} TechDocs',
+    'searches.title': 'Top recherches',
+    'searches.totalCount': '{{count}} recherches',
+    'searches.averagePrefix': 'En moyenne',
+    'searches.averageText': '{{count}} recherches par {{period}}',
+    'searches.averageSuffix': 'ont été effectuées durant cette période.',
+    'searches.hour': 'heure',
+    'searches.day': 'jour',
+    'users.title': "Nombre total d'utilisateurs",
+    'users.haveLoggedIn': 'se sont connectés',
+    'users.loggedInUsers': 'Utilisateurs connectés',
+    'users.licensed': 'Licenciés',
+    'users.ofTotal': 'sur {{total}}',
+    'users.tooltip':
+      "Définissez le nombre d'utilisateurs sous licence dans app-config.yaml",
+    'table.headers.name': 'Nom',
+    'table.headers.kind': 'Type',
+    'table.headers.lastUsed': 'Dernière utilisation',
+    'table.headers.views': 'Vues',
+    'table.headers.executions': 'Exécutions',
+    'table.headers.trend': 'Tendance',
+    'table.headers.entity': 'Entité',
+    'table.pagination.topN': 'Top {{count}}',
+    'filter.all': 'Tous',
+    'filter.selectKind': 'Sélectionner le type',
+    'common.noResults': 'Aucun résultat pour cette plage de dates.',
+    'common.readMore': 'Lire plus',
+    'common.exportCSV': 'Exporter CSV',
+    'common.downloading': 'Téléchargement...',
+    'common.today': "Aujourd'hui",
+    'common.yesterday': 'Hier',
+    'common.numberOfSearches': 'Nombre de recherches',
+    'common.filteredBy': 'filtré par',
+    'common.invalidDateFormat': 'Format de date invalide',
+    'permission.title': 'Permissions manquantes',
+    'permission.description':
+      "Pour voir le plugin Insights d'Adoption, contactez votre administrateur pour obtenir les permissions adoption-insights.events.read.",
```

Comment:
Similar comment as previously. Either keep the plugin name as is, like so:
```suggestion
      "Pour voir le plugin \"Adoption Insights\", contactez votre administrateur pour obtenir les permissions adoption-insights.events.read.",
```

Or translate:
```suggestion
      "Pour voir le plugin de statistiques d'adoption, contactez votre administrateur pour obtenir les permissions adoption-insights.events.read.",
```

Reply:
Updated.